tools/dm/skip-replace-sqls.md

```diff
@@ -0,0 +1,530 @@
+
+1. Use `sql-skip` or `sql-replace` to register an operator for the specified binlog position or DDL pattern.
+2. Use `resume-task` to resume the replication task.
+3. Regain and re-parse the binlog event that causes the replication error.
+4. The binlog event successfully matches with the registered operator in step 1.
+5. Execute the skip/replace operation corresponding to the operator and then the replication task continues.
+
+**Use `sql-skip` / `sql-replace` to preset operations to avoid breaking the replication**
+
+1. Use `sql-skip` or `sql-replace` to register an operator for the specified DDL pattern.
+2. Parse the relay log to obtain the binlog event.
+3. The binlog event (including the SQL statements unsupported by TiDB) successfully matches with the registered operator in step 1.
+4. Execute the skip/replace operation corresponding to the operator and then the replication task continues and will not get interrupted.
+
+**Use `sql-skip` / `sql-replace` to preset operations to avoid breaking the replication in the scenario of merging and replicating data from sharded tables**
+
+1. Use `sql-skip` or `sql-replace` to register an operator (on DM-master) for the specified DDL pattern.
+2. Each DM-worker parses the relay log to obtain the binlog event.
+3. DM-master coordinates the DDL lock replication among DM-workers.
+4. DM-master checks if the DDL lock replication succeeds, and sends the registered operator in step 1 to the DDL lock owner.
+5. DM-master requests the DDL lock owner to execute the DDL statement.
+6. The DDL statement that is to be executed by the DDL lock owner successfully matches with the received operator in step 4.
+7. Execute the skip/replace operation corresponding to the operator and then the replication task continues. 
+
+### Command
+
+When you use `dmctl` to manually handle the SQL statements unsupported by TiDB, the commonly used commands include `query-status`, `query-error`, `sql-skip` and `sql-replace`.
+
+#### query-status
+
+`query-status` allows you to query the current status of items such as the subtask and the relay unit in each DM-worker. For details, see [query status](/tools/dm/query-status.md).  
+
+#### query-error
+
+`query-error` allows you to query the existing errors of the running subtask and relay unit in DM-workers.
+
+##### Command usage
+
+```bash
+query-error [--worker=127.0.0.1:8262] [task-name]
+```
+
+##### Arguments description
+
++ `worker`:
+    - Flag parameter, string, `--worker`, optional
+    - If it is not specified, this command queries the errors in all DM-workers; if it is specified, this command queries the error of the specified DM-worker.
+
++ `task-name`:
+    - Non-flag parameter, string, optional
+    - If it is not specified, this command queries the errors of all subtasks; if it is specified, this command queries the error of the specified subtask.
+
+##### Example of results
+
+```bash
+» query-error test
+{
```

Comment:
According to the new code comment style, we need to add a period in the end for each comment, both phrases and complete sentences.